lib/Agrammon/DB/User.pm6

```diff
@@ -100,7 +100,7 @@ class Agrammon::DB::User does Agrammon::DB {
         return self;
     }
 
-    method load {
+    method load() {
```

Comment:
I'm a little curious why this was needed, tbh...they should mean the same thing. What did you observe?

Reply:
> I'm a little curious why this was needed, tbh...they should mean the same thing. What did you observe?

Without `()` after `load` I got in `t/user.t`:

<img width="710" alt="Screenshot 2021-02-08 at 15 19 02" src="https://user-images.githubusercontent.com/198038/107232020-3b857880-6a21-11eb-9ce7-51df72a5a1a6.png">

Reply:
Hm, that sounds like a bug. We'll investigate that.